xapian-applications/omega/templates/query

```diff
@@ -132,9 +141,9 @@ $SPAGE{$thispage}
 $map{$range{$add{$thispage,1},$opt{y}},$PAGE{$_}}
 $if{$ne{$opt{y},$lastpage},...}
```

Comment:
If you look at the output HTML, what I believe you're getting is something like (with different whitespace):

```
<ul class="pagination">
  <li>PREV LINK</li>
  <li>PAGE 1 LINK</li>
  ...
  </li>THIS PAGE</li>
```

You aren't allowed to put naked text inside a `ul` like that (although browsers will generally figure out a way of rendering it), and it's impossible to style. (It should also be `&hellip;`, really.)